Containers/fulltextsearch/Dockerfile

```diff
@@ -2,3 +2,5 @@
 FROM elasticsearch:7.17.6
 
 RUN elasticsearch-plugin install --batch ingest-attachment
+
+HEALTHCHECK CMD curl -skfI localhost:9200 || exit 1

```

Comment:
we don't need this as the base image already includes the healthcheck. See https://github.com/elastic/elasticsearch/blob/8b3293ddeb008459bf795cb4082beab3d654cd88/distribution/docker/src/docker/Dockerfile#L295

Reply:
![grafik](https://user-images.githubusercontent.com/75573284/186683074-5d6c41ee-e06d-43f7-a15e-5b789dd0b0d8.png)
I think it is needed

Reply:
Strange! Seems like you are right.

Reply:
![grafik](https://user-images.githubusercontent.com/75573284/186687881-cca9eaa2-eac4-4c3c-9c91-c47784de354a.png)
I think the reason is this if condition

Reply:
Can you add this back?